tests/integration/cloud/rest_contracts/conftest.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import annotations
+
+import enum
+import os
+import pathlib
+import subprocess
+from typing import TYPE_CHECKING, Callable, Final, Union
+
+import pytest
+from pact import Consumer, Pact, Provider
+from pact.matchers import Matcher  # noqa: TCH002
+from typing_extensions import Annotated  # noqa: TCH002
+
+from great_expectations.compatibility import pydantic
+from great_expectations.core.http import create_session
+
+if TYPE_CHECKING:
+    from requests import Session
+
+PACT_MOCK_HOST: Final[str] = "localhost"
+PACT_MOCK_PORT: Final[int] = 9292
+PACT_DIR: Final[str] = str(
+    pathlib.Path(pathlib.Path(__file__).parent, "pacts").resolve()
+)
+
+
+class RequestMethods(str, enum.Enum):
+    DELETE = "DELETE"
+    GET = "GET"
+    PATCH = "PATCH"
+    POST = "POST"
+    PUT = "PUT"
+
+
+@pytest.fixture
+def session() -> Session:
+    return create_session(access_token=os.environ.get("GX_CLOUD_ACCESS_TOKEN"))
+
+
+def get_git_commit_hash() -> str:
+    return subprocess.check_output(["git", "rev-parse", "HEAD"]).decode("ascii").strip()
+
+
+@pytest.fixture
+def pact(request) -> Pact:
+    pact_broker_base_url = "https://greatexpectations.pactflow.io"
+    consumer_name = "great_expectations"
+    provider_name = "mercury"
```

Comment:
Should these also be constants at the top of the file?

Reply:
I had them there initially, but they are only used in this one place. The other constants are used elsewhere. Maybe it's nice to just have them at the top? Not sure that it matters.

Reply:
If they aren't meant to be used outside this test, then I think keeping them in here is cleaner.
If they need to be reused, then we can move them.